### PR TITLE
mgr/rook: Add timezone info

### PR DESCRIPTION
**Pull request by varshar16**
Recently time zone info was added[1] to various properties like created,
started and others in cephadm and changes were made to orchestrator module too.
Let's add timezone info to rook as well, which will fix the type error.

[1] https://github.com/ceph/ceph/pull/37920

Fixes: https://tracker.ceph.com/issues/49126
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by jmolmo — APPROVED** [review]
LGTM

**Comment by varshar16** [comment]
jenkins test make check

**Comment by varshar16** [comment]
jenkins test api

File: src/pybind/mgr/rook/module.py [hunk 1]
```diff
@@ -263,7 +263,7 @@ def describe_service(self,
                          service_type: Optional[str] = None,
                          service_name: Optional[str] = None,
                          refresh: bool = False) -> List[orchestrator.ServiceDescription]:
-        now = datetime.datetime.utcnow()
+        now = datetime_now()
```
Comment (travisn) [inline src/pybind/mgr/rook/module.py:266]: does `datetime_now()` use UTC time?

Comment (varshar16) [reply src/pybind/mgr/rook/module.py:266]: Yes
https://github.com/ceph/ceph/blob/718af4faaf47c416544f872d2932fce52e60549b/src/python-common/ceph/utils.py#L5-L11

**Comment by varshar16** [comment]
jenkins test api

**Review by travisn — APPROVED** [review]

**Comment by varshar16** [comment]
ping @sebastian-philipp

**Review by sebastian-philipp — APPROVED** [review]

**Pull request merged**